pkgs/tools/security/hash-identifier/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitLab, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "hash-identifier";
+  version = "1.2";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/hash-identifier";
+    rev = "kali/${version}+git20180314-0kali1";
+    sha256 = "1amz48ijwjjkccg6gmdn3ffnyp2p52ksagy4m9gy8l2v5wj3j32h";
+  };
+
+  installPhase = ''
+    patchShebangs --host ./hash-id.py
+    mv ./hash-id.py ./hash-identifier
+    mkdir -p $out/bin
+    install -m775 -t $out/bin hash-identifier
+  '';
+
+  meta = with lib; {
+    description = "Software to identify the different types of hashes used to encrypt data and especially passwords.";
+    homepage = "https://gitlab.com/kalilinux/packages/hash-identifier";
```

We should point to the upstream's source repo.

```suggestion
    homepage = "https://github.com/blackploit/hash-identifier";
```

---

Sure.